pkg/operator/etc/constants.go

```diff
@@ -0,0 +1,23 @@
+package etc
+
+const (
+	// NamespaceName the name of the namespace in which Starboard-operator stores its
+	// configuration and where it runs scan jobs.
+	NamespaceName = "starboard-operator"
+
+	// ServiceAccountName the name of the service account used to provide
+	// identity for scan jobs run by Starboard-operator.
+	ServiceAccountName = "starboard-operator"
```

Comment:
Service account is configurable through ENV. Also these constants are duplication what we already have in the starboard package. Why do you need them?